pkg/fanal/types/handler.go

```diff
@@ -4,11 +4,12 @@ type HandlerType string
 
 const (
 	SystemFileFilteringPostHandler HandlerType = "system-file-filter"
+	SystemPackagesPostHandler      HandlerType = "system-packages"
 	UnpackagedPostHandler          HandlerType = "unpackaged"
 
 	// SystemFileFilteringPostHandlerPriority should be higher than other handlers.
 	// Otherwise, other handlers need to process unnecessary files.
 	SystemFileFilteringPostHandlerPriority = 100
-
-	UnpackagedPostHandlerPriority = 50
+	SystemPackagesPostHandlerPriority      = 75
```

Comment:
remove this?

Reply:
done at https://github.com/aquasecurity/trivy/pull/5439/commits/19fbf7b6f7d528e44d1cf720f78727c2956b4e27